### ISSUE TITLE
Port Bootstrap DSL to Java

### ISSUE THREAD

**Issue by aman-dureja** [CategoricalData/hydra#25]
https://github.com/CategoricalData/hydra/blob/main/hydra-haskell/src/main/haskell/Hydra/Dsl/Bootstrap.hs

**Comment by joshsh** [comment]
Note: the DSL modules do not need to be ported in a 1:1 fashion to Java -- the DSLs are free to make use of any language features which makes it convenient and idiomatic to create models and transformations in that particular language, and parity between languages is not necessary here. In this case, since the source of truth for Hydra Core is in Haskell, the bootstrap module might only be needed in Haskell.

**Comment by joshsh** [comment]
Indeed, the bootstrapping DSL is only needed in Haskell, which is the source of truth for the Hydra kernel. Closing.
